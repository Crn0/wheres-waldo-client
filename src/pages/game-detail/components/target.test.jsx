import { MemoryRouter } from 'react-router-dom';
import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import Target from './target';
import dataTest from '../data-test';

describe('Target', () => {
  it('renders the game information', async () => {
    const target = { ...dataTest.targets[0] };

    const { container } = render(
      <MemoryRouter>
        <Target target={target} />
      </MemoryRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
