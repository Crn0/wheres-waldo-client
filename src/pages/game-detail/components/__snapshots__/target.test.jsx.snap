// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Target > renders the game information 1`] = `
<div>
  <div
    class="_flex_a195c1 _items_center_a195c1 _gap_1_a195c1"
  >
    <div
      class="_img_target_a195c1"
    >
      <img
        alt="raft-man"
        src="https://res.cloudinary.com/dhtzg8kkq/image/upload/v1739172514/dev/wheres_waldo/dragon-island/raft-man_i1jqdl.png"
      />
    </div>
    <span>
      raft-man
    </span>
  </div>
</div>
`;
